cmd/operator/main.go

```diff
@@ -68,6 +69,7 @@ func init() {
 	flag.BoolVar(&analyticsEnabled, "analytics", true, "Send analytical event (Cluster Created/Deleted etc.) to Google Analytics")
 
 	flag.StringVar(&pvProvisioner, "pv-provisioner", constants.PVProvisionerGCEPD, "persistent volume provisioner type")
+	flag.StringVar(&storageClass, "storageclass", "", "Storage class type")
```

Comment:
operator probably should die with error message when both storage class and pv-provisioner are provided.

also this flag needs a better help message. storage class type is too generic.